### PR TITLE
Issue#152 login.html

### PR DESCRIPTION
**Pull request by lucas1004jx**
### Context: 
This PR is part of `Migration to es6+` #152.
_As this tasks is huge, we agreed to split it in little chunks so it can be review it faster_ 👍 

### PR Changelog
- change 'var' to 'const'
- change function to ES& arrow function 
- Simplify 'if' statement

### CODE REVIEW

File: app/views/login.html [hunk 1]
```diff
@@ -145,31 +145,28 @@
...
-    function areCookiesEnabled() {
-        var cookieEnabled = navigator.cookieEnabled;
+    const areCookiesEnabled = () => {
+        const cookieEnabled = navigator.cookieEnabled;
 
         // When cookieEnabled flag is present and false then cookies are disabled.
-        if (cookieEnabled === false) {
-            return false;
-        }
+        if (cookieEnabled === false) return false;
```
Comment (UlisesGascon) [inline app/views/login.html]: Maybe we can update this to `if (!cookieEnabled) return false`

**Comment by UlisesGascon** [comment]
Thanks for the PR @lucas1004jx! Welcome aboard :hugs:

I just added some context in the PR definition ;-)

File: app/views/login.html (hunk 1 above)
Comment (lucas1004jx) [reply app/views/login.html]: Yeah, good suggestion, I didn't notice that, thanks !

**Comment by ckarande** [comment]
@lucas1004jx Thanks for the PR and welcome to the project! Please coordinate with @UlisesGascon if you'd like to further assist on ES6+ migration on any other parts of the code.

**Comment by UlisesGascon** [comment]
As #160 was merged already... I will merge this one :-)

Thanks @lucas1004jx ! 🤗

**Pull request merged**